diff --git a/addon/fatfs/ffconf.h b/addon/fatfs/ffconf.h
index 9011a9a8..ecfd77ab 100644
--- a/addon/fatfs/ffconf.h
+++ b/addon/fatfs/ffconf.h
@@ -34,7 +34,7 @@
 /* This option switches f_mkfs(). (0:Disable or 1:Enable) */
 
 
-#define FF_USE_FASTSEEK	0
+#define FF_USE_FASTSEEK	1	/* mmbasic-fastseek (#984): with a kept-open FIL */
 /* This option switches fast seek feature. (0:Disable or 1:Enable) */
 
 
